packages/common/src/api/user.ts

```diff
@@ -193,13 +194,56 @@ const userApi = createApi({
     },
     getRemixersCount: {
       fetch: async (
-        { userId, trackId }: { userId: ID; trackId?: string },
+        { userId, trackId }: { userId: ID; trackId?: number },
         { audiusSdk }
       ) => {
         const sdk = await audiusSdk()
         const { data } = await sdk.full.users.getRemixersCount({
           id: Id.parse(userId),
-          trackId
+          userId: Id.parse(userId),
+          trackId: trackId ? encodeHashId(trackId) : undefined
+        })
+        return data
+      },
+      options: {}
+    },
+    getPurchasersCount: {
+      fetch: async (
+        {
+          userId,
+          contentId,
+          contentType
+        }: { userId: ID; contentId?: string; contentType: string },
+        { audiusSdk }
+      ) => {
+        const sdk = await audiusSdk()
+        const { data } = await sdk.full.users.getPurchasersCount({
+          id: encodeHashId(userId),
+          contentId: contentId ? Id.parse(contentId) : undefined,
```

Comment:
You can use `OptionalId.parse()` if the value might be missing. It doesn't throw in that case.